docs/collaboration.md

```diff
@@ -0,0 +1,68 @@
+# Collaborative Publishing with Posit Publisher
+
+This workflow guide explains how to collaborate with others on deployed content.
+
+## Configuration and Deployment files
+
+Posit Publisher creates and utilizes two `.toml` files for deploying content.
+Both are inside of the `.posit/publish` directory alongside your entrypoint
+file.
+
+- The Configuration file describes how the content will be deployed to Posit
+  Connect. It can be edited by hand and by interacting with the Posit
+  Publisher sidebar. It is named using the content's title and a unique
+  identifier, for example, `content-title-061E.toml`. See the
+  [Configuration file reference](./configuration.md) for a full list of
+  settings.
+- The Deployment file details where the content was deployed and its state.
+  Deployment files are automatically generated by Posit Publisher in the
+  `deployment` directory and updated during deploys. They are not meant to be
+  edited. Posit Publisher will take care of this file for you so you do not need
+  to think about it aside from the collaboration tips below.
+
+Utilizing both of these files, alongside a credential and collaborator access
+on Posit Connect, you and your collaborators can update and configure the same
+content. To grant others collaborator access see
+[Posit Connect User Guide- Access list settings](https://docs.posit.co/connect/user/content-settings/#set-collaborators).
```

Comment:
Is the space after the dash intended?

Reply:
Oops, there is a missing space there to match the other links. Great catch.